src/schema/types/payment.graphql

```diff
@@ -0,0 +1,25 @@
+# Types
+type Payment {
+    id: ID!
+    productId: String!
+    paymentId: Int!
+    status: String!
+    statusDetail: String!
+    dateApproved: Date!
+    paymentMethodId: String!
+    paymentTypeId: String!
+    payerEmail: String!
+}
+
+type PaymentInput {
```

Comment:
Let's rename it to `CheckoutInput` and change `type` by `input`:
```graphql
input CheckoutInput {
  userEmail: String! // to be later replaced by userId
  items: [LineItem!]!
  paymentMethod: CreditCardInput! // let's discuss offline about this
}
```

Reply:
```graphql
input LineItem {
  productId: ID
  amount: Int!
}
```

Reply:
```graphql
input CheckoutInput {
  productId: ID!
  paymentMethod: MercadoPagoInput!
}

enum PaymentMethodEnum {
  VISA
  MASTERCARD
  ...
}

input MercadoPagoInput {
  accountEmail: String!
  token: String!
  paymentMethod: PaymentMethodEnum!
}
```

Reply:
Done